build/yamls/base/conf/antrea-agent.conf

```diff
@@ -29,9 +29,9 @@ featureGates:
 # - stt
 #tunnelType: geneve
 
-# Default MTU to use for the host gateway interface and the network interface of each Pod. If
-# omitted, antrea-agent will default this value to 1450 to accommodate for tunnel encapsulate
-# overhead.
+# Default MTU to use for the host gateway interface and the network interface of each Pod.
+# If omitted, antrea-agent will discovery the MTU of the node's primary interface (e.g. eth0) and
```

Comment:
discovery -> discover
node -> Node

Probably remove (e.g. eth0) as in many cases the primary interface is not eth0.

Reply:
Done